### PR TITLE
Dance app

### PR DESCRIPTION
**Pull request by MiladNazeri**
PR for the marketplace Dance App

### CODE REVIEW

**Comment by salinacsoto** [comment]
Tested and QA Approved :)

File: Marketplace/DANCE/Dance-URLS.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    Script.resolvePath('./Animations/Thriller Part 2 566.fbx'),
+    Script.resolvePath('./Animations/Thriller Part 3 768.fbx'),
+    Script.resolvePath('./Animations/Thriller Part 4 1113.fbx'),
+    Script.resolvePath('./Animations/Twerk it Twerk it 457.fbx'),
+    Script.resolvePath('./Animations/Twista 284.fbx'),
+    Script.resolvePath('./Animations/Wavy baby 213.fbx'),
+    Script.resolvePath('./Animations/Ymca Dance 136.fbx')
+];
+
+module.exports = danceurls;
```
Comment (RebeccaStankus) [inline Marketplace/DANCE/Dance-URLS.js]: EOF line

File: Marketplace/DANCE/Animations/dance.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var fs = require("fs");
+var dir = fs.readdirSync(__dirname);
+var baseUrl = "https://hifi-content.s3.amazonaws.com/milad/ROLC/Organize/O_Projects/Hifi/Scripts/hifi-content/DomainContent/Rust/DanceApp/Animations/";
+var finalArray = dir.map(function(file){
+    return baseUrl + file;
+})
+console.log(finalArray);
```
Comment (RebeccaStankus) [inline Marketplace/DANCE/Animations/dance.js]: Did you want to leave this log? Also needs the EOF line...

File: Marketplace/DANCE/Tablet/js/DANCE_Tablet.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+                },
+                onBlur(){
+                    
+                    var sanitizedObject = Object.assign({}, this.dance, {
+                        startFrame: parseInt(Math.max(this.dance.startFrame, 0)),
+                        endFrame: parseInt(Math.min(this.dance.endFrame, this.maxEndFrame)),
+                        duration: parseInt(Math.max(1, this.dance.duration)),
+                        fps: parseInt(Math.min(this.dance.fps, 500))
+                    });
+                    console.log(JSON.stringify(sanitizedObject));
```
Comment (RebeccaStankus) [inline Marketplace/DANCE/Tablet/js/DANCE_Tablet.js]: Print statement

File: Marketplace/DANCE/dance.app.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+    "scriptURL": "http://mpassets.highfidelity.com/5665d68c-a7af-4b49-abc9-3bba51b4ec78-v1/DANCE/DANCE.js",
+    "homeURL": "http://mpassets.highfidelity.com/5665d68c-a7af-4b49-abc9-3bba51b4ec78-v1/DANCE/Tablet/DANCE_Tablet.html"
+}
```
Comment (RebeccaStankus) [inline Marketplace/DANCE/dance.app.json]: EOF...not sure if it's actually necessary here but Github is showing an error

File: Prototyping/HappyKiosk/Happy-Kiosk_Button_Server.js [hunk 5]
```diff
@@ -35,9 +35,9 @@
     // Init 
     var entityID,
         name,
-        position,
         kioskZoneID,
         self,
+        rotation,
```
Comment (RebeccaStankus) [inline Prototyping/HappyKiosk/Happy-Kiosk_Button_Server.js]: Should happy kiosk stuff be in here?

File: Prototyping/Improv-Stage-Manager/WebOverlay.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
+(function () {
```
Comment (RebeccaStankus) [inline Prototyping/Improv-Stage-Manager/WebOverlay.js]: Stage manager stuff?

**Pull request merged**